docs/robutekLibrary/Robutek-library/src/index.ts

```diff
@@ -0,0 +1,192 @@
+import { stdin } from "stdio";

```

Comment:
Prosím buďto o nahrazení nebo změnění tohoto zipu, aby to byl "prázdný základní projekt", tj. v `index.ts` bude jen to co je nutné na inicializaci robota, ready na to aby to děcka vzali jako počáteční šablonu a začali psát svůj program.